cmd/login/mc.go

```diff
@@ -83,8 +70,18 @@ func (r *runner) loginWithCodeName(ctx context.Context, codeName string) error {
 		fmt.Fprintf(r.stdout, "Switched to context '%s'.\n", contextName)
 	}
 
-	fmt.Fprint(r.stdout, color.GreenString("You are logged in to the management cluster of installation '%s'.\n", codeName))
+	return nil
+}
 
+// loginWithCodeName switches the active kubernetes context to
+// one with the name derived from the installation code name.
+func (r *runner) loginWithCodeName(ctx context.Context, codeName string) error {
```

Comment:
I switched the logic for `loginWithCodeName` with `loginWithKubeContextName` so that the first one calls the second and not the other way around as it used to be.